spec/controllers/showdates_controller_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+describe ShowdatesController do
+  before :each do
+    login_as_boxoffice_manager
+  end
+
+  context 'test strong params' do
+    before :each do
+      @show = Show.create!(listing_date: 1.month.ago, season: Time.this_season, name: 'Dummy show')
+      @mass_assignment_attack_params = {
+        showdate_type: 'Ts',
+        show_id: @show.id,
+        showdate: {
+          live_stream: '',
+          stream_anytime: '1',
+          description: '',
+          max_advance_sales: 50000,
+          show_id: @show.id,
+          access_instructions: 'idk dude ruby is hard for me too',
+          # these are 2 disallowed params that should cause an error
+          dummy_param_1: 'trying to sneak past',
+          dummy_param_2: 'sneaky uWu'
+        },
+        saved_max_sales: '',
+        day: [0],
+        time: {year: 2010, month: 1, day: 1, hour: 20, minute: 00},
+        show_run_dates: {start: '"2010-01-01"', end: '"2010-01-03"'},
+        stream_until: {year: 2010, month: 1, day: 1, hour: 23, minute: 45},
+      }
+      # dummy's to be filtered out
+      @dummies = [:dummy_param_1, :dummy_param_2]
+    end
+
+    context 'validate strong params' do
+
+      it 'checks strong params on create' do
```

Comment:
Perhaps, we should use more descriptive title. It seems like you are testing for the removal of attr_accessor, addition of strong_params, and create function. It might be good to separate them into three individual 'It'.